clink/app/src/loader/draw_test.cpp

```diff
@@ -13,7 +13,9 @@
 #include <terminal/printer.h>
 
 #include <getopt.h>
+#if defined(_M_IX86) || defined(_M_X64)
 #include <xmmintrin.h>
```

Comment:
The `#include <xmmintrin.h>` line can be deleted; it's unused on x86 and x64 as well.  I'll delete it after merging the PR.